doc/system_health_monitoring/system-health-HLD.md

```diff
@@ -156,12 +151,14 @@ Considering that different vendors platform may have different LED color capabil
 
 ## 2. System health monitor service business logic
 
-System health monitor daemon will running on the host, periodically(every 60s) check the "monit summary" command output and PSU, fan, thermal status which stored in the state DB, if anything wrong with the services monitored by monit or peripheral devices, system status LED will be set to fault status. When fault condition relieved, system status will be set to normal status. 
+System health monitor daemon will running on the host, periodically(every 60s) check critical services/processes status, the "monit summary" command output and PSU, fan, thermal status which stored in the state DB, if anything wrong with them, system status LED will be set to fault status. When fault condition relieved, system status will be set to normal status.
```

Comment:
`will running` ---> `will run`?

`, periodically(every 60s) check critical services/processes status, the "monit summary" command output and PSU, fan, thermal status which stored in the state DB,` ---> `and periodically (every 60 seconds) check critical services, processes status, output of the command "monit summary", PSU, Fan, and thermal status which is stored in the state DB.`?

Reply:
`if anything wrong with them` ---> `If anything is abnormal`?

How do we define the term `anything`?

Reply:
anything could be: services like swss, syncd; processes like orchagent; hardware like ASIC, PSU; and any other thing that is monitored by user defined checker. So, basically anything means that any object which is monitored by system health.